questionnaire/routes.js

```diff
@@ -107,4 +109,101 @@ router
         }
     });
 
+router
+    .route('/:questionnaireId/submissions')
+    .get(permissions('read:questionnaires'), async (req, res, next) => {
+        try {
+            const {questionnaireId} = req.params;
+            const questionnaireService = createQuestionnaireService({logger: req.log});
+            const result = await questionnaireService.getQuestionnaireSubmissionStatus(
+                questionnaireId
+            );
+            const submissionStatus = result.rows && result.rows[0].submission_status;
+            // the default value for this column is "NOT_STARTED", so if it doesn't
+            // exist, then it must not be a valid questionnaire ID.
+            if (!submissionStatus) {
+                const err = Error(
+                    `Questionnaire with questionnaireId "${questionnaireId}" does not exist`
+                );
+                err.name = 'HTTPError';
+                err.statusCode = 404;
+                err.error = '404 Not Found';
+                throw err;
+            }
+
+            const response = await questionnaireService.getSubmissionResponseData({
+                questionnaireId,
+                submissionStatus
+            });
+
+            if (response.data.attributes.submitted) {
+                await questionnaireService.updateQuestionnaireSubmissionStatus(
+                    questionnaireId,
+                    'COMPLETED'
+                );
+                response.data.attributes.status = 'COMPLETED';
+                await questionnaireService.submitQuestionnaire(questionnaireId);
+            }
+
+            // // const notificationResponse = {body: 'Message sent'};
```

Comment:
Can you stick a TODO on these comments if you're coming back to them? If not, just delete them